### PR TITLE
Add documentation about the Qiskit Code Assistant

### LINKED ISSUE #1930: Add documentation for the Qiskit Code Assistant
### URL, if applicable

https://docs.quantum.ibm.com/guides

### Describe the new content you are requesting.

Introduce documentation about the Qiskit Code Assistant: what is it, how to use it, etc.

### If this new content request is accepted, do you want to write the content?

I will write a draft of the proposed content

### PR DESCRIPTION
**Pull request by cbjuan**
1st draft for documentation about the Qiskit Code Assistant. Should fix #1930 

Considerations:
- Rough draft, feel free to edit or give any feedback
- I'm not sure if we should split the documentation in 2 parts: one for JupyterLab extension and another one for the VSCode extension
- I'm not sure the where the Qiskit Code Assistant should be placed in the TOC
- I added content that depends on #1918

### CODE REVIEW

File: docs/guides/qiskit-code-assistant.mdx [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+title: Qiskit Code Assistant
+description: Improve your quantum computing developer experience and learn best practices for Qiskit and IBM Quantum Platform services with Qiskit Code Assistant
+---
+
+# Qiskit Code Assistant
+
+Improve your quantum computing developer experience and learn best practices for Qiskit and IBM Quantum Platform services with Qiskit Code Assistant!
```
Comment (javabster) [inline docs/guides/qiskit-code-assistant.mdx:8]: This first paragraph sounds a bit like a marketing advert, could you adjust the tone a bit to be more documentation style?

File: docs/guides/qiskit-code-assistant.mdx [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+After installing, you will need to reload the extension/IDE (typically VSCode will ask you to do it automatically). When ready, you will see the following in the lower part of the user interface.
+
+![../\_images/run/qiskit\-code\-assistant\vscode\-selected\-model.png](/images/run/qiskit-code-assistant/vscode-selected-model.png)
+
+
+## First steps using the Qiskit Code Assistant
+
+### Authentication and setup
+
+After installing the extension in VSCode or JupyterLab, the extension will try to authenticate you. By default, the package tries to authenticate to IBM Quantum services with the defined Qiskit API token, and uses your token from the `QISKIT_IBM_TOKEN` environment variable or from the file `~/.qiskit/qiskit-ibm.json` (under the section `default-ibm-quantum`).
```
Comment (javabster) [inline docs/guides/qiskit-code-assistant.mdx:70]: do we need to explain to people how to set their api token env var?

File: docs/guides/qiskit-code-assistant.mdx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+In addition, once the completer runs you can use the buttons on the inline completer
+widget to cycle or accept
+
+![../\_images/run/qiskit\-code\-assistant\jupyterlab-inline-complete.png](/images/run/qiskit-code-assistant/jupyterlab-inline-complete.png)
+
+> NOTE: The service can sometime take a few seconds to retrun a suggestion, you can see when the service is working by checking the status bar
+
+Jupyterlab also includes a traditional completer than displays suggestion in a context menu rather than inline. This completer can be triggered using `Tab` to run and display the context menu.
+
+The context menu will include suggestions from the default completer in addition to the Qiskit Code Assistant suggestions. The context menu also sanitizes and trims the suggestions, making it less useful for see the code suggestion before inserting it.
```
Comment (javabster) [inline docs/guides/qiskit-code-assistant.mdx:103]: typos?

**Review by javabster — COMMENTED** [review]
Generally looks good! Just a few comments.

Question: do you think it would be better to split the content into 2 pages? one for the vscode extension and one for the jupyter lab extension? I felt like the page is quite long while i was reading and if a reader is only interested in one of the methods theres a lot of redundant info

File: docs/guides/qiskit-code-assistant.mdx [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+If you need help to install or run JupyterLab, please refer to the [JupyterLab documentation](https://jupyterlab.readthedocs.io/en/latest/)
+
+### Installing the VSCode extension
+
+To install the [VSCode extension](https://github.com/Qiskit/qiskit-code-assistant-vscode), you can add it to your IDE as follows:
+
+1. Launch Visual Studio Code
+1. Click on the Extensions tab
+1. Search for `qiskit-vscode`
+1. Click **Install** for the "Qiskit Code Assistant"
```
Comment (beckykd) [inline docs/guides/qiskit-code-assistant.mdx:57]:
```suggestion
1. Find "Qiskit Code Assistant" and click **Install**.
```

**Review by beckykd — COMMENTED** [review]
Thanks for this draft.  I'll finish my review tomorrow!

**Comment by frankharkins** [comment]
> ...do you think it would be better to split the content into 2 pages? one for the vscode extension and one for the jupyter lab extension?

I think the tabs component could work quite well here

**Comment by cbjuan** [comment]
> > ...do you think it would be better to split the content into 2 pages? one for the vscode extension and one for the jupyter lab extension?
> 
> I think the tabs component could work quite well here

Splitting in tabs

**Comment by cbjuan** [comment]
Now the guide uses tabs: one giving an overview about the assistant, other two tabs for the extensions. Addressed some other comments and added some videos about how to use it

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 5]
```diff
@@ -0,0 +1,105 @@
+---
+title: Use the Qiskit Code Assistant in JupyterLab
+description: How to install, use, and configure the Qiskit Code Assistant in JupyterLab.
+---
+
+
+# Use the Qiskit Code Assistant in JupyterLab
```
Comment (javabster) [inline docs/guides/qiskit-code-assistant-vscode.mdx:7]: This intro looks like its for the jupyter lab page not the vscode page?

**Comment by cbjuan** [comment]
I’ll stop editing the files unless you ask for it so I don’t interrupt others

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+### Authentication and setup
+
+After installing the extension, it tries to authenticate you. By default, the package tries to authenticate to IBM Quantum services with the defined Qiskit API token, and uses your token from the `QISKIT_IBM_TOKEN` environment variable or from the file `~/.qiskit/qiskit-ibm.json` (under the section `default-ibm-quantum`). If you need help configuring your account, follow the instructions in [Set up to use IBM Quantum Platform.](/guides/setup-channel#set-up-to-use-ibm-quantum-platform)
+
+By default, the extension uses the `granite-8b-qiskit` model, which is listed in the Model Picker in the bottom status bar.
+
+![The bottom status bar is shown, with `granite-8b-qiskit` listed.](/images/guides/qiskit-code-assistant/vscode-selected-model.png)
+
+The first time you use the `granite-8b-qiskit` model a modal opens, listing some major restrictions that you should be aware of when using the model. Clicking `Accept` will accept the disclaimer and enable the model for code generation.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:57]:
```suggestion
The first time you use the `granite-8b-qiskit` model, a modal opens listing some major restrictions that you should be aware of when using the model. Clicking `Accept` will accept the disclaimer and enable the model for code generation.
```

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+![The window that opens upon first use contains restrictions tha must be accepted.](/images/guides/qiskit-code-assistant/eula.png)
+
+### Generate Code
+After accepting the EULA you can open a new file and start coding. To get a code suggestion, type `Ctrl+.` as shown:
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:62]:
```suggestion
After accepting the EULA, you can open a new file and start coding. To get a code suggestion, type `Ctrl` + `.` as shown:
```

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+![A code suggestion is shown.  The suggested code is in grey.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion.png)
+
+Press `Tab` to accept the model-generated suggestion or press `ESC` to reject it.
+
+<Admonition type="tip" title="Tips for working with code suggestions">
+- Type `Ctrl+.` again to generate additional suggestions.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:69]:
```suggestion
- Type `Ctrl` + `.` again to generate additional suggestions.
```

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+- When a suggestion is shown, you can use `Alt/Option+]` and `Alt/Option+[` to cycle forward and backward, respectively, through the suggestions (if available).
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:70]:
```suggestion
- When a suggestion is shown, you can use `Alt/Option` + `]` and `Alt/Option + `[` to cycle forward and backward, respectively, through the suggestions (if available).
```

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+## Troubleshooting
+
+If you are not seeing the extension status bar in VSCode, check that the extension is installed and enabled under the extensions tab.
+
+If the extension is installed and enabled, but cannot select a model, verify that your current API Token has been added and the `Qiskit Code Assistant: Url` is properly set.
+
+## Contribute to the JupyterLab extension
+
+The code for this official extension is publicly available and open source. Check it out in [github.](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab)
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:93]: This should be for VSCode, right?

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Check out examples to use the Qiskit Code Assistant for [circuits](https://docs.quantum.ibm.com/guides/circuit-library), [configuring error suppression](https://docs.quantum.ibm.com/guides/configure-error-suppression), and [transpiling with passmanagers.](transpile-with-pass-managers)
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:99]: This won't be love on Monday morning. Should we remove this section for now?

**Review by Eric-Arellano — COMMENTED** [review]
(still reviewing)

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+1. Click **Uninstall** for the "Qiskit Code Assistant".
+
+
+## Troubleshooting
+
+If you are not seeing the extension status bar in VSCode, check that the extension is installed and enabled under the extensions tab.
+
+If the extension is installed and enabled, but cannot select a model, verify that your current API Token has been added and the `Qiskit Code Assistant: Url` is properly set.
+
+## Contribute to the JupyterLab extension
```
Comment (beckykd) [inline docs/guides/qiskit-code-assistant-vscode.mdx:91]:
```suggestion
## Contribute to the VSCode extension
```

File: docs/guides/qiskit-code-assistant-vscode.mdx (hunk 10 above)
Comment (beckykd) [inline docs/guides/qiskit-code-assistant-vscode.mdx:93]:
```suggestion
The code for this official extension is publicly available and open source. Check it out in [github.](https://github.com/Qiskit/qiskit-code-assistant-vscode)
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+
+<Admonition type="note" title="Notes">
+    - This is an experimental feature available only to IBM Quantum&trade; Premium Plan users.
+    - The Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+
+<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum, the Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user ask to the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and get the desired output." className="max-w-auto h-auto" controls>
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:17]:
```suggestion
<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum, the Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user asks the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and gets the desired output." className="max-w-auto h-auto" controls>
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+  <source src="/videos/guides/qiskit-code-assistant/demo-JupyterLab-t1t2-granite-8b-qiskit-short.mp4" type="video/mp4"></source>
+</video>
+
+## Install the JupyterLab extension
+To install the [JupyterLab extension](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab), run the following command from a terminal:
+```sh
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:23]:
```suggestion

```sh
```

File: docs/guides/qiskit-code-assistant-vscode.mdx (hunk 11 above)
Comment (beckykd) [reply docs/guides/qiskit-code-assistant-vscode.mdx:99]: Yep

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+
+```sh
+jupyter lab
+```
+
+The extension loads automatically and is listed on the bottom of the JupyterLab window.
+
+![Screenshot of the bar on the bottom of JupyterLab.](/images/guides/qiskit-code-assistant/jupyterlab-selected-model.png)
+
+For help working with JupyterLab, refer to the [JupyterLab documentation.](https://jupyterlab.readthedocs.io/en/latest/)
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:37]: Iirc, you prefer this style?

```suggestion
Refer to the [JupyterLab documentation.](https://jupyterlab.readthedocs.io/en/latest/) for help working with JupyterLab
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+
+## Configure extension settings
+
+It is recommended that you edit the following JupyterLab settings by going Settings > Settings Editor:
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:41]: Not sure if you agree with the arrow suggestion

```suggestion
It is recommended that you edit the following JupyterLab settings by going to Settings -> Settings Editor:
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 17]
```diff
@@ -0,0 +1,133 @@
...
+
+- Click **Inline Completer**, find "Show widget" and choose **Always**.  This means that the the inline completer widget will always be shown so you can cycle through and select a completion item.
+
+- Click **Code Completion** and increase the value for "Default timeout for a provider." to `10000` or 10 seconds. The default value is 1 second, but the Qiskit Code Assistant API might take longer than this to find a suggstion. This setting only applies to the standard context menu that is invoked with `Tab`. The inline completer has a default of 10 seconds.
+
+Other settings you might want to change:
+
+- Keyboard shortcuts can be changed from Settings > Settings Editor > Keyboard Shortcuts.
+
+- You can change the IBM Quantum API token to use in the JupyterLab command palette. To do that, type `Alt + Shift + C`, search for `qiskit`, select the **Qiskit Code Assistant: Set IBM Quantum API token** command, and paste in your token.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:51]:
```suggestion
- You can change the IBM Quantum API token to use in the JupyterLab command palette. To do that, type `Alt` + `Shift` + `C`, search for `qiskit`, select the **Qiskit Code Assistant: Set IBM Quantum API token** command, and paste in your token.
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 18]
```diff
@@ -0,0 +1,133 @@
...
+
+- [Advanced] To change the instance of the Qiskit Code Assistant Service that the extension should use, edit the Qiskit Code Assistant `serviceUrl` setting.
+
+- [Advanced] Keyboard shortcuts can be changed by searching for `completer` in the Keyboard Shortcuts settings (Settings > Settings Editor > Keyboard Shortcuts) and adding new shortcuts for the relevant commands.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:55]: Feel free to disagree

```suggestion
- [Advanced] Keyboard shortcuts can be changed by searching for `completer` in the Keyboard Shortcuts settings (Settings -> Settings Editor -> Keyboard Shortcuts) and adding new shortcuts for the relevant commands.
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 19]
```diff
@@ -0,0 +1,133 @@
...
+
+### Generate Code
+
+While you develop your code using Qiskit, you can ask to the Qiskit Code Assistant to help you. In general, the assistant suggests better code as response to Python comments or docstrings, but you can use the assistant anywhere in your file.
+
+In JupyterLab, there are two ways to use the code autocompletion: inline or in a context menu.
+
+Inline autocomplettion can be triggered by using the following key strokes:
+
+- `Alt .` or `Alt \` runs autocomplete at the current cursor location.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:79]:
```suggestion
- `Alt` + `.` or `Alt` + `\` runs autocomplete at the current cursor location.
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 20]
```diff
@@ -0,0 +1,133 @@
...
+- `Alt [` and `Alt ]` can be used to cycle through the list of suggestions if there are
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:80]:
```suggestion
- `Alt` + `[` and `Alt` + `]` can be used to cycle through the list of suggestions if there are
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 21]
```diff
@@ -0,0 +1,133 @@
...
+  more than one.
+- `Alt Tab` or `Alt END` accepts the suggested code and inserts it at the current
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:82]:
```suggestion
- `Alt` + `Tab` or `Alt` + `END` accepts the suggested code and inserts it at the current
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 22]
```diff
@@ -0,0 +1,133 @@
...
+
+Additionally, after the assistant runs, you can use the buttons on the widget to cycle or accept the suggestion:
+
+![The popup pane with buttons is shown.](/images/guides/qiskit-code-assistant/jupyterlab-inline-complete.png)
+
+<Admonition type="note">
+The service can sometimes take a few seconds to return a suggestion, you can see when the service is working by checking the status bar.
+</Admonition>
+
+Jupyterlab also includes a traditional suggestion context menu. Use `Tab` to run and display the context menu.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:93]:
```suggestion
Jupyterlab also includes a traditional suggestion context menu. Use the `Tab` key to run and display the context menu.
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 23]
```diff
@@ -0,0 +1,133 @@
...
+extension, check the frontend extension is installed:
+
+```bash
+jupyter labextension list
+```
+
+
+## Contribute to the JupyterLab extension
+
+The code for this extension is publicly available and open source. Check it out in [github.](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab)
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:127]:
```suggestion
The code for this extension is publicly available and open source. Check it out in [GitHub.](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab)
```

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx [hunk 24]
```diff
@@ -0,0 +1,133 @@
...
+
+## Next steps
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-jupyterlab.mdx:129]: Ditto if this should be removed for now?

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 25]
```diff
@@ -0,0 +1,100 @@
...
+The first time you use the `granite-8b-qiskit` model, a modal opens listing some major restrictions that you should be aware of when using the model. Clicking `Accept` will accept the disclaimer and enable the model for code generation.
+
+![The window that opens upon first use contains restrictions tha must be accepted.](/images/guides/qiskit-code-assistant/eula.png)
+
+### Generate Code
+After accepting the EULA, you can open a new file and start coding. To get a code suggestion, type `Ctrl` + `.` as shown:
+
+![A code suggestion is shown.  The suggested code is in grey.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion.png)
+
+Press `Tab` to accept the model-generated suggestion or press `ESC` to reject it.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:66]:
```suggestion
Press the `Tab` key to accept the model-generated suggestion or press the `ESC` key to reject it.
```

File: docs/guides/qiskit-code-assistant-vscode.mdx [hunk 26]
```diff
@@ -0,0 +1,100 @@
...
+Learn how to install, use, configure and uninstall the official Qiskit Code Assistant extension in Visual Studio Code (VSCode).
+
+<Admonition type="note" title="Notes">
+    - This is an experimental feature available only to IBM Quantum&trade; Premium Plan users.
+    - The Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+
+<video title="A user uses the Qiskit Code Assistant in Visual Studio Code. The user has previous imports and a function definition called get_random_linear_function with a docstring defining what the function should do. After typing Ctrl+. the Qiskit Code Assistant autocompletes the function with the code to generate a random linear function using Numpy and Qiskit. The video ends with the user executing the code generated and getting a matrix representing a random linear function in the terminal." className="max-w-auto h-auto" controls>
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant-vscode.mdx:17]:
```suggestion
<video title="A user uses the Qiskit Code Assistant in Visual Studio Code. The user has previous imports and a function definition called get_random_linear_function, with a docstring defining what the function should do. After typing the Ctrl and . keys, the Qiskit Code Assistant autocompletes the function with the code to generate a random linear function using Numpy and Qiskit. The video ends with the user executing the code generated and getting a matrix representing a random linear function in the terminal." className="max-w-auto h-auto" controls>
```

File: docs/guides/qiskit-code-assistant.mdx [hunk 27]
```diff
@@ -0,0 +1,76 @@
+---
+title: Qiskit Code Assistant
+description: Learn how to use Qiskit Code Assistant, a generative AI code assistant.
+---
+
+# Qiskit Code Assistant
+
+The Qiskit Code Assistant aims to make quantum computing more accessible to new Qiskit adopters, and improve the coding experience for current users. It is a generative AI code assistant powered by [watsonx](https://www.ibm.com/products/watsonx-ai). It is trained using millions of text tokens from Qiskit SDK v1.x, years of Qiskit code examples and IBM Quantum&trade; features. The Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite 8B Code,](https://www.ibm.com/products/watsonx-ai/foundation-models) which incorporate the latest features and functionalities from IBM&reg;.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant.mdx:8]:
```suggestion
The Qiskit Code Assistant aims to make quantum computing more accessible to new Qiskit adopters and to improve the coding experience for current users. It is a generative AI code assistant powered by [watsonx](https://www.ibm.com/products/watsonx-ai). It is trained using millions of text tokens from Qiskit SDK v1.x, years of Qiskit code examples and IBM Quantum&trade; features. The Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite 8B Code,](https://www.ibm.com/products/watsonx-ai/foundation-models) which incorporate the latest features and functionalities from IBM&reg;.
```

File: docs/guides/qiskit-code-assistant.mdx [hunk 28]
```diff
@@ -0,0 +1,76 @@
...
+
+<Admonition type="note" title="Notes">
+    - This is an experimental feature available only to IBM Quantum Premium Plan users.
+    - The Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum, the Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user ask to the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and get the desired output." className="max-w-auto h-auto" controls>
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant.mdx:16]:
```suggestion
<video title="A user uses the Qiskit Code Assistant in JupyterLab. The user has a comment in the code to get the available backends for their account in IBM Quantum. The Qiskit Code Assistant gives a code suggestion to get the available backends. The user executes the code, gets the available backends in IBM Quantum Platform for the current account. The user ask to the Qiskit Code Assistant to generate the code to get the T1, T2 and other properties for the qubit 1 in the device ibm_brisbane. The assistant gives a code suggestion. The user runs the suggested code and gets the desired output." className="max-w-auto h-auto" controls>
```

File: docs/guides/qiskit-code-assistant.mdx [hunk 29]
```diff
@@ -0,0 +1,76 @@
...
+  <source src="/videos/guides/qiskit-code-assistant/demo-JupyterLab-t1t2-granite-8b-qiskit-short.mp4" type="video/mp4"></source>
+</video>
+
+## Features
+
+* Accelerates Qiskit code generation by leveraging generative AI based on the `granite-8b-qiskit` model.
+* Use abstract and specific prompts to generate recommendations.
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant.mdx:23]: The conjugation is inconsistent here. Ditto rest of list

File: docs/guides/qiskit-code-assistant-jupyterlab.mdx (hunk 18 above)
Comment (beckykd) [reply docs/guides/qiskit-code-assistant-jupyterlab.mdx:55]: The dash is fine.  In my old tool, typing > added a little arrow, so this was just forgetfulness.

File: docs/guides/qiskit-code-assistant.mdx [hunk 30]
```diff
@@ -0,0 +1,76 @@
...
+Year = {2024},
+Eprint = {arXiv:2406.14712},
+}
+```
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Install and use the official [JupyterLab](/guides/qiskit-code-assistant-jupyterlab) or [VSCode](/guides/qiskit-code-assistant-vscode) extensions.
+    - Follow examples to use the Qiskit Code Assistant for [circuits](https://docs.quantum.ibm.com/guides/circuit-library), [configuring error suppression](https://docs.quantum.ibm.com/guides/configure-error-suppression), and [transpiling with passmanagers.](transpile-with-pass-managers)
```
Comment (Eric-Arellano) [inline docs/guides/qiskit-code-assistant.mdx]: This should be removed for now

File: docs/guides/qiskit-code-assistant.mdx (hunk 29 above)
Comment (beckykd) [reply docs/guides/qiskit-code-assistant.mdx:23]: yeahhhh - it's ugly because they're not all really the same type of activity.  Let me see if I can fix.

**Review by Eric-Arellano — APPROVED** [review]

**Comment by cbjuan** [comment]
Thank you!! 🙏

**Pull request merged**